README.md

```diff
@@ -1,4 +1,15 @@
-usage : tty-clock [-iuvsScbtrahDBxn] [-C [0-7]] [-f format] [-d delay] [-a nsdelay] [-T tty] 
+## Instalation
+```
+cd /tmp
+wget https://github.com/xorg62/tty-clock/archive/master.zip
```

Comment:
there are releases now - not sure how to get the latest version? maybe tell people to use git instead...